### PR TITLE
Get posts and get post routers

### PR DESCRIPTION
**Pull request by Mohamed-Shafei20**

### CODE REVIEW

File: src/routes/get-all.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Router } from "express";
+const router = Router();
```
Comment (aim97) [inline src/routes/get-all.ts]: Remove definition from imports

File: src/routes/get-all.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import { posts } from "../posts";
+
+router.get("/api/posts", (req, res) => {
+  if (posts.length === 0) {
```
Comment (aim97) [inline src/routes/get-all.ts:5]: The spider returning posts should return an empty array if there are no posts to return.

**Review by aim97 — APPROVED** [review]
Good job.

File: src/routes/get-all.ts (hunk 1 above)
Comment (Mohamed-Shafei20) [reply src/routes/get-all.ts]: done

File: src/routes/get-all.ts (hunk 2 above)
Comment (Mohamed-Shafei20) [reply src/routes/get-all.ts:5]: done

**Pull request merged**